docs/source/conf.py

```diff
@@ -264,15 +264,15 @@
 # 'pointsize': '10pt',
 
 # Additional stuff for the LaTeX preamble.
-# 'preamble': '',
+#  'preamble': '',
 }
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
     ('index', 'poliastro.tex', 'poliastro Documentation',
-   'Juan Luis Cano Rodríguez', 'manual'),
+        'Juan Luis Cano Rodríguez', 'manual'),
```

Comment:
Why?

Reply:
The flake8 always shows an error here & above in the commented `LaTex` part. I made the suggested changes & voila! here we are again :thinking:

Reply:
Weird. In any case, let's get it working and don't pay too much attention to it.